src/util/classes.ts

```diff
@@ -23,26 +24,68 @@ export interface Category {
   children?: Category[];
 }
 
-const COLOR_UNCAT = '#CCC';
+// https://colorhunt.co/palette/4802
+const COLOR_GREEN = '#96cd39',
+  COLOR_SUPER_GREEN = '#54e346',
+  COLOR_BRIGHT_GREEN = '#A8FC00',
+  COLOR_UNCAT = '#CCC',
+  COLOR_SLIGHTLY_YELLOW = '#f5ff65',
+  COLOR_ORANGE = '#ffba47',
+  COLOR_RED = '#ff5b44';
 
 // The default categories
 // Should be run through createMissingParents before being used in most cases.
 export const defaultCategories: Category[] = [
   {
-    name: ['Work'],
-    rule: { type: 'regex', regex: 'Google Docs|libreoffice|ReText' },
-    data: { color: '#0F0' },
+    name: ['Work', 'Writing'],
+    rule: { type: 'regex', regex: 'Google Docs|Google Sheets|libreoffice|ReText' },
+    data: { color: COLOR_GREEN },
+  },
+  {
+    name: ['Work', 'Writing'],
+    rule: {
+      type: 'regex',
+      select_keys: ['app'],
+      regex: 'LibreOffice|TextEdit|MacDown|Obsidian|TextEdit',
+    },
+    data: { color: COLOR_GREEN },
+  },
+  {
+    name: ['Work', 'General'],
+    rule: {
+      type: 'regex',
+      regex: 'Preview|Finder|Todoist|1Password|Soulver|System Preferences|VNC Viewer|Streaks',
+      select_keys: ['app'],
+    },
+    data: { color: COLOR_SLIGHTLY_YELLOW },
   },
   {
     name: ['Work', 'Programming'],
     rule: {
       type: 'regex',
-      regex: 'GitHub|Stack Overflow|BitBucket|Gitlab|vim|Spyder|kate|Ghidra|Scite',
+      regex:
+        'GitHub|Stack Overflow|BitBucket|Gitlab|vim|Spyder|kate|iTerm|Hyper|Alacritty|Script Editor|Ghidra|Scite',
```

Comment:
please consider adding `jetbrains-pycharm` and `jetbrains-idea` to this list

Reply:
@soxofaan are these the application names on macos? Doesn't seem like it. Make a [suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) and I'll include it in this PR.